src/nanoemoji/nanoemoji.py

```diff
@@ -89,15 +87,40 @@ def _bool_flag(name):
     return flag
 
 
-def write_preamble(nw):
-    def module_rule(mod_name, arg_pattern, rspfile=None, rspfile_content=None):
+def _ufo_rule(master: MasterConfig) -> str:
+    return "write_" + master.style_name.lower() + "_ufo"
```

Comment:
could `master.style_name` contain spaces? Should we not `.replace(" ", "")` them? Or will the ninja writer automatically escape them?